src/test/html/dom-module-name-or-is_test.ts

```diff
@@ -18,11 +18,12 @@ import {Analyzer} from 'polymer-analyzer';
 
 import {Linter} from '../../linter';
 import {registry} from '../../registry';
-import {WarningPrettyPrinter} from '../util';
+import {assertExpectedFixes, WarningPrettyPrinter} from '../util';
 
 const fixtures_dir = path.join(__dirname, '..', '..', '..', 'test');
+const ruleId = 'dom-module-name-or-is';
```

Comment:
Hm, looks like we've got an inconsistency in the name of the file vs the id of the rule. Mind moving files around to standardize on `dom-module-name-or-is`?

Would involve using ruleId in the getRules call in setup, and renaming the ts file and test directory